### ISSUE TITLE
unable to debug using Go Nightly

### ISSUE THREAD

**Issue by OneOfOne** [golang/vscode-go#107]
### What version of Go, VS Code & VS Code Go extension are you using?
- Run `go version` to get version of Go
	- `go version devel +6bf2eea62a 2020-05-28 14:43:19 +0000 linux/amd64`
- Run `code -v` or `code-insiders -v` to get version of VS Code or VS Code Insiders
	- `1.45.1` (and latest inside as of 10 minutes ago as well)
- Check your installed extensions to get the version of the VS Code Go extension 
	- works: `0.14.3`, doesn't work: `2020.5.2722`
- Run `go env` to get the go development environment details
```sh
GO111MODULE=""
GOARCH="amd64"
GOBIN="/home/oneofone/code/go/bin"
GOCACHE="/home/oneofone/.cache/go-build"
GOENV="/home/oneofone/.config/go/env"
GOEXE=""
GOFLAGS="-gcflags=-c=16"
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GOMODCACHE="/home/oneofone/code/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/oneofone/code/go"
GOPRIVATE=""
GOPROXY="direct"
GOROOT="/usr/src/go"
GOSUMDB="off"
GOTMPDIR=""
GOTOOLDIR="/usr/src/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build413442895=/tmp/go-build -gno-record-gcc-switches"
```

### Share the Go related settings you have added/edited

```json
	"go.buildOnSave": "off",
	"go.vetOnSave": "off",
	"go.lintOnSave": "off",
	"go.useCodeSnippetsOnFunctionSuggest": true,
	"go.useCodeSnippetsOnFunctionSuggestWithoutType": true,
	"go.autocompleteUnimportedPackages": true,
	"go.lintTool": "staticcheck",
	"go.editorContextMenuCommands": {
		"toggleTestFile": true,
		"addTags": true,
		"removeTags": true,
		"testAtCursor": false,
		"testFile": false,
		"testPackage": true,
		"generateTestForFunction": true,
		"generateTestForFile": false,
		"generateTestForPackage": true,
		"addImport": false,
		"testCoverage": true,
		"playground": false
	},
	"go.addTags": {
		"tags": "json",
		"options": "json=omitempty",
		"promptForTags": false,
		"transform": "camelcase"
	},
	"go.coverOnSingleTest": true,
	"go.gotoSymbol.includeGoroot": true,
	"go.useLanguageServer": true,
	"go.languageServerExperimentalFeatures": {
		"format": true,
		"diagnostics": true,
		"documentLink": true
	},
	"go.delveConfig": {
		"dlvLoadConfig": {
			"followPointers": true,
			"maxVariableRecurse": 1,
			"maxStringLen": 64,
			"maxArrayValues": 64,
			"maxStructFields": -1
		},
		"apiVersion": 2,
		"showGlobalVariables": true
	},
	"gopls": {
		"env": {},
		"staticcheck": true,
		"analyses": {
			"ST1000": false,
			"ST1003": false
		},
		"codelens": {
			"generate": true,
			"upgrade.dependency": true,
			"test": true
		},
		"usePlaceholders": false,
		"completeUnimported": true,
		"deepCompletion": true
	},
```

### Describe the bug
Using this `launch.json` config:
```json
{
	"version": "0.2.0",
	"configurations": [
		{
			"name": "Launch",
			"type": "go",
			"request": "launch",
			"mode": "auto",
			"program": "${workspaceFolder}",
			"env": {},
			"trace": "verbose",
			"args": [""]
		}
	]
}
```

I can't start debugging with the nightly version, same config and settings and everything works perfectly fine with the previous version.

### Steps to reproduce the behavior:
1. Press F5 to start the debugger.
2. Debug bar shows for a second then disappears.
3. Debug Console output is empty.
4. Go debug channel is empty.
5. The only log that changes is `Log (Window)` and it shows:

```
[2020-05-28 15:05:39.020] [renderer1] [error] timeout after 1000 ms: Error: timeout after 1000 ms
    at t.RawDebugSession.handleErrorResponse (file:///usr/src/VSCode-linux-x64/resources/app/out/vs/workbench/workbench.desktop.main.js:3061:355)
    at file:///usr/src/VSCode-linux-x64/resources/app/out/vs/workbench/workbench.desktop.main.js:3060:679
    at async t.RawDebugSession.shutdown (file:///usr/src/VSCode-linux-x64/resources/app/out/vs/workbench/workbench.desktop.main.js:3058:772)
```

**Comment by stamblerre** [comment]
Thanks for the report!

/cc @quoctruong

**Comment by hyangah** [comment]
The fix is included in `2020.6.204` released today. 
Please test it. Thanks!!!

**Comment by OneOfOne** [comment]
Tested it and everything seems to work now, I'll open a new ticket if I run into any bugs, thanks.
